### PR TITLE
Prevent App Nap

### PR DESCRIPTION
**Pull request by kevgo**
first naive attempt at implementing #64, description in the README changes

@charlierudolph

### CODE REVIEW

File: src/index.ls [hunk 1]
```diff
@@ -61,12 +64,15 @@ Tertestrial = new Liftoff name: 'tertestrial', config-name: 'tertestrial', exten
...
         console.log '\nrunning'
+        spinner.start! if process.env.TERTESTRIAL_PREVENT_APP_NAP
 
     chokidar.watch(env.config-path).on 'change', (path) ->
+      spinner.stop!
       reset-terminal!
       console.log 'Reloading configuration\n'
       config := new ConfigFile env.config-path
       command-runner.update-config config
+      spinner.start! if process.env.TERTESTRIAL_PREVENT_APP_NAP
```
Comment (charlierudolph) [inline src/index.ls]: Why do you need to stop and start when the configuration changes?

**Comment by dmh43** [comment]
Is it tertestrial itself that is being put to sleep or the terminal app? There seems to be a way to disable app nap for a specific application, but it's not shown to me as an option for either my terminal application, or for the tertestrial bin file.

I haven't run into this issue yet, but I just upgraded to the latest macOS, so maybe I will soon...

**Comment by kevgo** [comment]
@dmh43 there used to be an option in Mavericks and El Capitan. It never allowed to co figure Terminal, though. And it's gone in Sierra (thanks Apple!). 

I don't know whether it's the whole Terminal or just a background process inside the terminal that's getting suspended. But outputting something once in a while seems to help here.

File: src/index.ls [hunk 2]
```diff
@@ -61,12 +64,15 @@ Tertestrial = new Liftoff name: 'tertestrial', config-name: 'tertestrial', exten
         else
           console.log "to exit, run #{cyan 'fg'}, then hit #{bold '[ctrl-c]'}\n"
         console.log '\nrunning'
+        spinner.start! if process.env.TERTESTRIAL_PREVENT_APP_NAP
```
Comment (kevgo) [inline src/index.ls]: Okay, done

File: src/index.ls (hunk 1 above)
Comment (kevgo) [inline src/index.ls]: That was a left-over when I thought configuring this via the config file was a good idea  :)  Removed.

**Comment by kevgo** [comment]
All comments addressed

**Pull request closed without merging**